README.md

```diff
@@ -1,4 +1,53 @@
+# Development process
+This section describes tools that help you with coding. We provide scripts that allows to 
+clone all repositories and install artifacts to Maven local / Maven Central Snapshot repositories.
+
+## Prerequisites
+```
+$>git clone git@github.com:Knotx/knotx-aggregator.git
+$>chmod -R 755 knotx-aggregator/**/*.sh
```

Comment:
How about [`git update-index`](https://git-scm.com/docs/git-update-index) ?
You may set proper rights directly in the git repository.
e.g. `git update-index --add --chmod=+x *.sh`